packages/libra-rpc-graphql/src/client.ts

```diff
@@ -73,9 +85,17 @@ export function createLibraRpc(target: string) {
   };
 }
 
+export function getFaucetUri(target: string) {
+  if (target.startsWith('http')) {
+    return target;
+  }
+
+  return KnownFaucets[target];
+}
+
 export function createLibraFaucet(target: string) {
+  const baseUri = getFaucetUri(target);
```

Comment:
there should only ever be a testnet faucet, but who knows what will happen in the future.